### PR TITLE
add totals visualisation

### PR DESCRIPTION
**Pull request by Tim-Abwao**
## Description

- Add visualizations: a line-plot of totals, bar-plot of confirmed cases by country, and geographical scatter-plot of cased in Africa. These are saved as png files in the `images` folder, and can be displayed on the repository landing page.
- Reschedule the workflow file to update the archive and create graphs only once a day; for a single commit a day. The geographical scatter-plot file added changes on each run, and triggers 2 commits for the twice-a-day schedule.

Fixes # (issue)
    #4 

## Type of change

- [x] New feature (non-breaking change which adds functionality)
    Adding visualizations

## Screenshots
lineplots

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by 4bic** [comment]
Hi @Tim-Abwao , this is fantastic, ⭐ 

a slight update,  It'd be nice to have the `png` files in an `img / images`  folder

File: README.md [hunk 1]
```diff
@@ -2,8 +2,11 @@
 
 `covid19-in-africa` is a dataset repository for COVID-19 cases in Africa. We Load data from Johns Hopkins University Center for Systems Science and Engineering [(JHU CSSE) GitHub](https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data).
 
+![africa totals](datasets/africa_totals.png)
```
Comment (4bic) [inline README.md:5]: save to folder

File: README.md [hunk 2]
```diff
@@ -2,8 +2,11 @@
...
 
+![africa totals](datasets/africa_daily.png)
```
Comment (4bic) [inline README.md:7]: ditto

**Review by 4bic — CHANGES_REQUESTED** [review]

**Comment by Tim-Abwao** [comment]
Great idea, @4bic 
Working on it.

**Pull request merged**